app/qml/FeaturePanel.qml

```diff
@@ -260,6 +260,9 @@ Drawer {
         }
     }
 
-    ExternalResourceBundle {id: externalResourceBundle}
+    ExternalResourceBundle {
+      id: externalResourceBundle
+      customPrefix: __merginApi.username
+    }
```

Comment:
Hmm I would skip using mergin usernames in the panel. It's adding something unrelated - people may not use mergin at all...  My preference is not to have username added to image file paths - having timestamps should be good enough to avoid conflicts.

Reply:
Agree that timestamp should be good enough to avoid conflicts. However, similar approach with using username/ID has been mentioned in issue description and therefore has been implemented. If a user is not log into Mergin, only timestamp will be used anyway. 

I am easy with either keeping or removing custom prefix.